app/main.py

```diff
@@ -2,0 +1,51 @@
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None :
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None :
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str):
+        super().__init__(name)
+        for skill in ["JavaScript", "HTML", "CSS"]:
+            self.learn_skill(skill)
+
+    def create_awesome_web_page(self) -> str:
+        print(f"{self.name} is creating a webpage...")
+        return "<h1>Hello world</h1>"
+
+
+class BackendDeveloper(FrontendDeveloper):
+    def __init__(self, name: str) -> None :
+        super().__init__(name)
+        # self.skills.clear()
```

Comment:
The line `# self.skills.clear()` is commented out. If the intention is to clear the skills list when initializing a `BackendDeveloper`, consider uncommenting this line to ensure the skills are reset.